dapps/digital-twin-detail/src/components/container/DataSetForm.ts

```diff
@@ -103,7 +103,7 @@ export default class DataSetFormComponent extends mixins(EvanComponent) {
   static isControlRequired(controlOpts: EvanFormControlOptions, type: string, subSchema: any): boolean {
     // check if min value is set required flag
     return type === 'array' || type === 'object' || type === 'boolean'
-      || (ajvMinProperties[type] && !Number.isNaN(subSchema[ajvMinProperties[type]]));
+      || (ajvMinProperties[type] && subSchema[ajvMinProperties[type]]) > 0;
```

Comment:
What did this fix?

Reply:
in case of e.g. type === "string" there is no value set for `subSchema[ajvMinProperties[type]]` but `ajvMinProperties[type]` exists.
Results in `!Number.isNaN(undefined)` which is true, meaning field is required, but it's not.
I also guess a min length of `0` is the same as not required.

Reply:
But yeah I didn't a deep dive in AJV validation, if there is a better way to do this I'm open to change it.